src/components/Onboarding/steps/SetPassword.js

```diff
@@ -61,6 +65,8 @@ class SetPassword extends PureComponent<Props, State> {
     const { nextStep, saveSettings } = this.props
 
     await db.setEncryptionKey('app', 'accounts', newPassword)
+    await changeLibcorePassword('', newPassword)
+    setLibcorePassword(newPassword)
```

Comment:
which we don't await?

Reply:
Same as before, I didn't want to keep the user hanging while waiting for libcore to reload to go to the next step :/